az-digital/az_quickstart
# Fixes #2257, Fixes #2258 Publication Person Block Title, Add Publication Taxonomy

## Description
This PR adds a taxonomy for publications, along with an associated entity reference field on publications for categorizing them. A contextual filter is also added to both publication view displays to allow filtering the views by publication category.

Permissions are added that allow content administrators to manage the publication categories.

Additionally, the `Hide block if the view output is empty` option is turned on for the Publication Person block. This prevents the Person block from being shown if the person on which the block is placed has no publications.

## Related issues
#2257 
#2258 

## How to test

- enable `az_publication`
- Create Publication Category terms
- Create Publications tagged with terms
- Assign `AZ Publications: Person` block to Person content pages
- Visit a person linked with an author that has publications assigned, verify the publications block appears
- Visit a person either not linked with an author or an author that has no publications assigned, verify the block does not appear
- Create a flexible page and add the publications search display as a view reference
- Verify working as expected
- Edit the view reference to contain arguments for a citation style and a taxonomy term id number argument, e.g. `az_ieee/14` or `az_ieee/14+52`
- Verify the citation style is used on the search page, and only the selected categories appear

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
